.devcontainer/Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:24.04
+
+LABEL "name"="Statick DevContainer"
+LABEL "version"="0.0"
+LABEL "repository"="https://github.com/sscpac/statick"
+LABEL "homepage"="https://github.com/sscpac/statick"
```

Comment:
These should be changed.  I'm about to open a PR that updates URL references project-wide.

Reply:
[b8a0d66](https://github.com/triboelectric/statick/pull/534/commits/b8a0d663e9c83ebc311058e693976c8f023bfa8d)